### DISCUSSION TITLE
Provide option for pagination text to be hidden or visible

### DISCUSSION THREAD

**Discussion by amoktar** [livewire/livewire#8008] [3. Feature ideas]
Hello Livewire team
I am happy to suggest this idea, and I hope you find it acceptable
It will be great if I could control weather to show or hide the text that comes with pagination `Showing` , `of`  and so.

```php
{{ $posts->links(data: ['text' => false]) }}
```

Thanks for great work

**Comment by joshhanley** [comment]
@AMoktar you can publish the pagination views using `php artisan livewire:publish --pagination` and update them to suit your needs 🙂

Hope this helps!

**Reply by amoktar** [reply]
Thanks @joshhanley I appreciate your help

**Reply by amoktar** [reply]
I'm working on a pr I hope I can do it right

**Reply by joshhanley** [reply]
@AMoktar No worries! 🙂 if you get stuck feel free to tag me and I can see if I can help.
